flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/nodes/CygnusApplication.java

```diff
@@ -134,6 +138,12 @@ public static void main(String[] args) {
                 mgmtIfPort = new Integer(commandLine.getOptionValue('p')).intValue();
             } // if
             
+            int pollingTime = 30; // default value
```

Comment:
Fixed in commit https://github.com/telefonicaid/fiware-connectors/commit/bf7dd2244880d3915955fbd5e9e7857483d26cfc